### DISCUSSION TITLE
Need help with custom theme

### DISCUSSION THREAD

**Discussion by v-Nyo** [Taiko2k/Tauon#1040] [Q&A]
Hi, I have some issues with my custom theme. Can't figure out how to theme some elements:

![393923](https://github.com/Taiko2k/TauonMusicBox/assets/135179386/e0803a32-9b6d-4c05-947b-56b48595ead8)
![935923534](https://github.com/Taiko2k/TauonMusicBox/assets/135179386/f9541188-8c5d-4ab8-bad4-8e9c8c6318f2)

https://github.com/v-Nyo/tauon-nord

Not matter what I change colors seem to be stuck with the ones from the ash theme (I changed every single color)

Help would be really appreciated (also feel free to leave some feedback :)

**Comment by 1tituz** [comment]
I guess you can't change the color of all elements.
-> https://github.com/Taiko2k/TauonMusicBox/wiki/Theming#notes

**Reply by 1tituz** [reply]
Hmm, i don't have this issue with my themes. 
I assumed these colors are generated automatically.
